src/profile/Index.vue

```diff
@@ -42,7 +42,7 @@
       </v-col>
 
       <v-col cols="12" sm="6" md="4">
-        <U2fCard :meta="mfa.u2f"/>
+        <SecurityKeyCard :meta="mfa.webauthn"/>
```

Comment:
So does this mean that if someone still has a U2F record but not WebAuthn, it won't look like they have any Security Key?

Reply:
Possibly, Do you see a better way of doing this?

@fillup was thinking of maybe converting all u2f to webauthn in the database, but I'm not sure if he's decided about this.